pkgs/applications/display-managers/lemurs/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  fetchFromGitHub,
+  lib,
+  linux-pam,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lemurs";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "coastalwhite";
+    repo = pname;
```

Comment:
`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21